gradle/libs.versions.toml

```diff
@@ -0,0 +1,40 @@
+[versions]
+activity-compose = "1.8.0"
+autoservice = "1.1.1"
+autoserviceKsp = "1.1.0"
+kotlin = "1.9.10"
+ksp = "1.9.10-1.0.13"
+versionsPlugin = "0.49.0"
+kotlinPoet = "1.14.2"
+agp = "8.1.2"
+org-jetbrains-kotlin-android = "1.9.10"
+junit = "4.13.2"
+compileTesting = "1.5.0"
+
+[libraries]
+autoservice-ksp = { module = "dev.zacsweers.autoservice:auto-service-ksp", version.ref = "autoserviceKsp" }
+autoservice-annotations = { module = "com.google.auto.service:auto-service-annotations", version.ref = "autoservice" }
+android-runtime = "com.google.android:android:4.1.1.4"
+android-gradlePlugin = { module = "com.android.tools.build:gradle", version.ref = "agp" }
+junit = "junit:junit:4.13.2"
+kotlinPoet-core = { module = "com.squareup:kotlinpoet", version.ref = "kotlinPoet" }
+kotlinPoet-ksp = { module = "com.squareup:kotlinpoet-ksp", version.ref = "kotlinPoet" }
+kotlin-stdlib = { module = "org.jetbrains.kotlin:kotlin-stdlib-jdk7", version.ref = "kotlin" }
+ksp = { module = "com.google.devtools.ksp:symbol-processing-api ", version.ref = "ksp" }
+plugin-ksp = { module = "com.google.devtools.ksp:com.google.devtools.ksp.gradle.plugin", version.ref = "ksp" }
+junit-junit = { group = "junit", name = "junit", version.ref = "junit" }
+truth = { module ="com.google.truth:truth", version = "1.1.4" }
```

Comment:
low logaf, but i've found just having all the versions up top helps for simplicity even if its only used once.